.yarn/patches/nx-npm-18.3.5-58c0bedf91.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/src/command-line/release/changelog.js b/src/command-line/release/changelog.js
+index f997f4c45e841bcdce86c9ffc6e776c08ccdbb60..ed9cbdcfa0913b1a4662eb35bd98526e1e863869 100644
+--- a/src/command-line/release/changelog.js
++++ b/src/command-line/release/changelog.js
+@@ -364,15 +364,20 @@ function resolveChangelogRenderer(changelogRendererPath) {
+     const interpolatedChangelogRendererPath = (0, utils_1.interpolate)(changelogRendererPath, {
+         workspaceRoot: workspace_root_1.workspaceRoot,
+     });
++    const path = require("node:path");
++    const joinedChangelogRendererPath = path.join(
++      workspace_root_1.workspaceRoot,
++      interpolatedChangelogRendererPath
++    );
+     // Try and load the provided (or default) changelog renderer
+     let changelogRenderer;
+     let cleanupTranspiler = () => { };
+     try {
+-        const rootTsconfigPath = (0, typescript_1.getRootTsConfigPath)();
+-        if (rootTsconfigPath) {
+-            cleanupTranspiler = (0, register_1.registerTsProject)(rootTsconfigPath);
+-        }
+-        const r = require(interpolatedChangelogRendererPath);
++        // const rootTsconfigPath = (0, typescript_1.getRootTsConfigPath)();
++        // if (rootTsconfigPath) {
++        //     cleanupTranspiler = (0, register_1.registerTsProject)(rootTsconfigPath);
```

Comment:
I think just commenting the one line that calls registerTsProject is enough

Reply:
Because it's a `require()` call, we need the full JS path:

```plaintext
Cannot find module 'tools/release/changelog-renderer'
  Require stack:
  - /Users/josh/repos/typescript-eslint/node_modules/nx/src/command-line/release/changelog.js
  - /Users/josh/repos/typescript-eslint/node_modules/nx/src/command-line/release/index.js
  - /Users/josh/repos/typescript-eslint/node_modules/nx/release/index.js
  - ...
```